keyboards/planck/keymaps/vaire/keymap.c

```diff
@@ -0,0 +1,254 @@
+    TO(0),      KC_MENU,    	KC_TRNS,        KC_TRNS,        KC_LALT,        KC_LCTL,        KC_TRNS,     	KC_TRNS,    	KC_TRNS,    	KC_LEFT,        KC_DOWN,   	KC_RCTL),
+
+  KEYMAP(
+    KC_BSPC,    KC_P,       	KC_O,           KC_I,           KC_U,           KC_Y,           KC_T,       	KC_R,       	KC_E,       	KC_W,           KC_Q,            KC_BSPC, 
+    KC_TAB,     KC_SCLN,        KC_L,           KC_K,           KC_J,           KC_H,           KC_G,       	KC_F,       	KC_D,       	KC_S,           KC_A,            KC_QUOT, 
+    KC_LSFT,    KC_ENT,         KC_DOT,    	    KC_COMM,   	    KC_M,           KC_N,           KC_B,       	KC_V,       	KC_C,		   KC_X,         	KC_Z,            SFT_T(KC_ENT), 
+    TO(0),      KC_LCTL,        KC_LALT,        KC_LGUI,        MO(2),          KC_TRNS,   	    KC_TRNS,     	MO(3),      	KC_SLSH,    	KC_LEFT,        KC_DOWN,	     LCTL(KC_RGHT)),
+ 
+    
+
+};
+
+
+
+
+#ifdef AUDIO_ENABLE
+
+
+#define ZELDA_PUZZLE \
+    Q__NOTE(_G5),     \
+    Q__NOTE(_FS5),    \
+    Q__NOTE(_DS5),     \
+    Q__NOTE(_A4),    \
+    Q__NOTE(_GS4),     \
+    Q__NOTE(_E5),     \
+    Q__NOTE(_GS5),     \
+    HD_NOTE(_C6),
+
+#define KC_ZELDA_TREASURE \
+    Q__NOTE(_A4 ), \
+    Q__NOTE(_AS4), \
+    Q__NOTE(_B4 ), \
+    HD_NOTE(_C5 ), \
+
+#define MARIO_THEME \
+    Q__NOTE(_E5), \
+    H__NOTE(_E5), \
+    H__NOTE(_E5), \
+    Q__NOTE(_C5), \
+    H__NOTE(_E5), \
+    W__NOTE(_G5), \
+    Q__NOTE(_G4),
+
+#define MARIO_GAMEOVER \
+    HD_NOTE(_C5 ), \
+    HD_NOTE(_G4 ), \
+    H__NOTE(_E4 ), \
+    H__NOTE(_A4 ), \
+    H__NOTE(_B4 ), \
+    H__NOTE(_A4 ), \
+    H__NOTE(_AF4), \
+    H__NOTE(_BF4), \
+    H__NOTE(_AF4), \
+    WD_NOTE(_G4 ),
+
+#define SONIC_RING \
+    E__NOTE(_E6),  \
+    E__NOTE(_G6),  \
+    HD_NOTE(_C7),
+
+    #define IMPERIAL_MARCH \
+  HD_NOTE(_A4), HD_NOTE(_A4), HD_NOTE(_A4), QD_NOTE(_F4), QD_NOTE(_C5), \
+  HD_NOTE(_A4), QD_NOTE(_F4),  QD_NOTE(_C5), WD_NOTE(_A4), \
+  HD_NOTE(_E5), HD_NOTE(_E5), HD_NOTE(_E5), QD_NOTE(_F5), QD_NOTE(_C5), \
+  HD_NOTE(_A4), QD_NOTE(_F4),  QD_NOTE(_C5), WD_NOTE(_A4)
+
+  #define ONE_UP_SOUND \
+    Q__NOTE(_E6  ),  \
+    Q__NOTE(_G6  ),  \
+    Q__NOTE(_E7  ),  \
+    Q__NOTE(_C7  ),  \
+    Q__NOTE(_D7  ),  \
+    Q__NOTE(_G7  ),
+
+    #define NUMBER_ONE \
+    HD_NOTE(_F4 ), \
+    Q__NOTE(_C5 ), \
+    E__NOTE(_B4 ), \
+    E__NOTE(_C5 ), \
+    E__NOTE(_B4 ), \
+    E__NOTE(_C5 ), \
+    Q__NOTE(_B4 ), \
+    Q__NOTE(_C5 ), \
+    H__NOTE(_AF4), \
+    HD_NOTE(_F4 ), \
+    Q__NOTE(_F4 ), \
+    Q__NOTE(_AF4), \
+    Q__NOTE(_C5 ), \
+    H__NOTE(_DF5), \
+    H__NOTE(_AF4), \
+    H__NOTE(_DF5), \
+    H__NOTE(_EF5), \
+    Q__NOTE(_C5 ), \
+    Q__NOTE(_DF5), \
+    Q__NOTE(_C5 ), \
+    Q__NOTE(_DF5), \
+    H__NOTE(_C5 ),
+
+float tone_my_startup[][2] = SONG(ZELDA_PUZZLE);
```

Comment:
A lot of these defines should be in your config.h file, not here: 
https://docs.qmk.fm/#/feature_audio

The lock keys should be fine though.